aws-java-sdk-sts/pom.xml

```diff
@@ -16,7 +16,7 @@
     <dependency>
       <groupId>com.amazonaws</groupId>
       <artifactId>aws-java-sdk-core</artifactId>
-      <version>1.10.57</version>
+      <version>1.10.58-SNAPSHOT</version>
```

Comment:
Yeah this is annoying, I wouldn't worry about it too much. We will actually overwrite this when we release so do whatever allows you to test this.